plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -207,6 +207,18 @@ export type CreatePermissionIntegrationRouterResourceOptions<
   ) => Promise<Array<TResource | undefined>>;
 };
 
+/**
+ * Options for creating a permission integration router
+ *
+ * @public
+ */
+export type OptionResources<TResourceType extends string, TResource> = {
+  resources:
+    | { permissions: Array<Permission> }
```

Comment:
Is it the case that these should be constrained to `BasicPermission`, while the one under the resource options are `ResourcePermission` of the same resource type?

It's still a bit unclear to me why we'd want the `{ resources: { permissions: ... } }` alternative, could you elaborate on why we wouldn't handle that through the top-level `permissions` option?

Reply:
yep, this wasn't needed.  Fixed in https://github.com/backstage/backstage/pull/17077/commits/b947f4230d49e825333bc802fa5a1530f18faa1e
now `resources` takes only an array

Reply:
Could one want to pass `{ permissions: ..., resources: ... }` as well?

Reply:
yeah good point, this is still a possibility.
Added support here: https://github.com/backstage/backstage/pull/17077/commits/4dd6bbe59e6073f89fe188b5fa0b8578016d6686